### ISSUE TITLE
:lady_beetle: pdagent enqueue fails with "=," when using -f 

### ISSUE THREAD

**Issue by NargiT** [PagerDuty/go-pdagent#46]
This is the command we use to trigger a notification from nagios to pagerduty

```bash
pdagent enqueue --routing-key XXX --event-action trigger --source "Nagios/test_host" --dedup-key `date "+%s"` --summary foobar --severity "critical" -f SERVICEOUTPUT='$LONGSERVICEOUTPUT$'
```

if I replace `$LONGSERVICEOUTPUT$` with
- `'=,'` -> invalid argument "SERVICEOUTPUT==," for "-f, --field" flag:  must be formatted as key=value
- `',,='` -> invalid argument "SERVICEOUTPUT=,,=" for "-f, --field" flag:  must be formatted as key=value
- `',=,'` -> invalid argument "SERVICEOUTPUT=,=," for "-f, --field" flag:  must be formatted as key=value
- `'="'` -> invalid argument "SERVICEOUTPUT==\"" for "-f, --field" flag: parse error on line 1, column 15: bare " in non-quoted-field

we get the following error, this fails no matter if there are values between quotes and the equals sign must be there

**Comment by DanHosier** [comment]
I'm seeing  potentially the same issue and can providing a minimal real-world example
```
$ pdagent nagios enqueue -f SERVICEOUTPUT="10,000 - 9,999 = 1"                # 2 commas and one equal sign
invalid argument "SERVICEOUTPUT=10,000 - 9,999 = 1" for "-f, --field" flag: 000 - 9 must be formatted as key=value

# Two examples that pass the field parsing:
$ pdagent nagios enqueue -f SERVICEOUTPUT="1,000 - 9999 = 1"                  # only 1 comma and one equal sign
$ pdagent nagios enqueue -f SERVICEOUTPUT="10,000 - 9,999 equals 1"           # 2 commas, and s/=/equals/

```

**Comment by finalcoz** [comment]
I am having the same problem. Since the pdagent nagios integration reports all alerts as high urgency, while we want warnings or some-nonproduction alerts to route to low urgency, we have custom code that gets called by nagios commands, and then relays the alerts with pdagent enqueue. I've had our custom code replace '=' in service output and a couple of other fields as a work around to this issue.

In investigating, I could not find different parts of the error message I receive from pdagent (like the error DanHosier listed) in the go-pdagent code, so I suspect that it may be passing along an error from the PD API.

(P.S. I have seen no changes to the code in years, other than automated MD file changes, so this project is looking quite dead.)

**Comment by NargiT** [comment]
@finalcoz yes the project is dead and the hilarious part is that the current pdagent is also dead. Right now pagerduty just do not care about nagios anymore.

<img width="844" height="133" alt="Image" src="https://github.com/user-attachments/assets/f6b9d6e7-537a-4100-9a7f-37c74dd5aad4" />
